src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -342,6 +342,10 @@ public static function create(string $uri, string $method = 'GET', array $parame
         $server['REQUEST_METHOD'] = strtoupper($method);
 
         $components = parse_url($uri);
+        if ($components === FALSE) {
+            trigger_deprecation('symfony/http-foundation', '6.3', 'Calling "%s()" without an invalid URI is deprecated.', __METHOD__ );
```

Comment:
typo in the message. Calling it **with** an invalid URI is what is deprecated.

Reply:
thanks, fixed.